modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/sys/SqlSystemTable.java

```diff
@@ -159,7 +159,7 @@ public SqlSystemTable(CreateTableData data, SqlSystemView view) {
 
     /** {@inheritDoc} */
     @Override public TableType getTableType() {
-        return TableType.SYSTEM_TABLE;
+        return TableType.VIEW;
```

Comment:
Now class name `SqlSystemTable` looks confusing. Actually it is an _adapter_ of our _system view_ to be understandable by H2 (as an H2 `Table`). Can we rename it accordingly to something like `SystemViewH2Adapter`? If yes, attention should be also put to `SqlSystemTableEngine`.

Reply:
yes, agree.